pytestqt/qtbot.py

```diff
@@ -234,7 +234,7 @@ def stopForInteraction(self):
 
     stop = stopForInteraction
 
-    def waitSignal(self, signal=None, timeout=1000, raising=None):
+    def waitSignal(self, signal=None, timeout=1000, callback=None, raising=None):
```

Comment:
Also I feel `callback` is to generic... perhaps `check_params_callback` would convey the intent a little better?